src/components/ArticlePage.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import api from "../utils/api";
+import {
+  BsFillArrowUpCircleFill,
+  BsFillArrowDownCircleFill,
+} from "react-icons/bs";
+
+function ArticlePage() {
+  let { article_id } = useParams();
+  const [article, setArticle] = useState({});
+
+  const {
+    article_img_url,
+    author,
+    body,
+    comment_count,
+    created_at,
+    title,
+    topic,
+    votes,
+  } = article;
+
+  useEffect(() => {
+    const fetchArticle = async () => {
+      const response = await api.get(`/articles/${article_id}`);
+      setArticle(response.data.article);
+    };
+
+    fetchArticle();
+  }, [article_id]);
+
+  return (
```

Comment:
With regarts to DRY-ing this code up. I would suggest extracting some of this into separage re-useable components.